code/src/index.css

```diff
@@ -11,3 +13,96 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.submitBox {
+background-color: rgb(242, 241, 241);
+}
+
+.submitForm {
+  display: flex;
+  flex-direction: column;
+  width: 95%;
+  margin: 0 auto;
+  }
+
```

Comment:
I LOVED that you challenged your self with Flexbox this week, i think thats smart :) (and i will learn grid from you this week 😍 )